package.json

```diff
@@ -68,6 +69,7 @@
     "fancy-test": "^2.0.16",
     "globby": "^11.1.0",
     "husky": "6",
+    "lodash": "^4.17.21",
```

Comment:
We'd like to avoid adding libraries when not necessary, even dev deps.  Can you please change the test so that it doesn't need `_.set`?  We'll also need to get approval for `cli-truncate` which should be fine.